scalafix-core/src/main/scala/scalafix/internal/patch/PatchInternals.scala

```diff
@@ -154,7 +154,7 @@ object PatchInternals {
       .mkString
   }
 
-  private def underlying(patch: Patch): Seq[Patch] = {
+  def underlying(patch: Patch): Seq[Patch] = {
```

Comment:
Why removing this `private` modifier?

Reply:
to use it in test and show that it will create two patches

Reply:
Then I suggest you use a package private modifier:

```suggestion
  private[scalafix] def underlying(patch: Patch): Seq[Patch] = {
```